config/runtime/tests/watchdog-reset.jinja2

```diff
@@ -0,0 +1,12 @@
+- test:
+    timeout:
+      minutes: {{ test_timeout|default('2') }}
```

Comment:
I thought of making this timeout configurable because the sleep duration for the test might vary (based on the watchdog timeout) and the action timeout needs to be set accordingly. On second thought, I think this can be a fixed overarching timeout. I'll fix this.